lxc/action.go

```diff
@@ -239,13 +239,8 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 			return err
 		}
 
-		// "start" for a frozen instance means "unfreeze"
```

Comment:
This was just a cleanup since it's not strictly needed with new servers anymore; I don't think an API extension is needed (at least for this situation), since the server will behave the same with and without this change. I'll drop this commit.